### PR TITLE
Few cleanup fixes

### PR DESCRIPTION
**Pull request by anoopcs9**
* Removes an extra forward slash **_/_**
* Replace **_bases_** with **_resources_** for _kustomize_. **_bases_** field was [deprecated](https://kubectl.docs.kubernetes.io/blog/2019/06/18/v2.1.0/#resources-expanded-bases-deprecated) long time back in favour of **_resources_**.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -53,7 +53,7 @@ endif
 GOARCH?=$(shell $(GO_CMD) env GOARCH)
 
 # Local (alternative) GOBIN for auxiliary build tools
-GOBIN_ALT:=$(CURDIR)/.bin/
+GOBIN_ALT:=$(CURDIR)/.bin
```
Comment (phlogistonjohn) [inline Makefile:56]: This change seems unrelated to the kustomize/kubectl deprecation of "bases". If this is a useful change I'd like to see it split out and described, thanks.

**Review by phlogistonjohn — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (anoopcs9) [reply Makefile:56]: Done.

**Review by phlogistonjohn — APPROVED** [review]
LGTM, thanks!
